source/manual/readmes.html.md

```diff
@@ -7,105 +7,73 @@ type: learn
 parent: "/manual.html"
 ---
 
-This is a guide to writing and maintaining README documents for GOV.UK's public repositories.
+This is a guide to writing and maintaining README documents for GOV.UK's public repositories. It's based on [guidance we have in the GDS Way for READMEs](https://gds-way.cloudapps.digital/manuals/readme-guidance.html#writing-readmes).
 
-## Who are READMEs for (aka what's the user need)?
+READMEs are for a technical audience. This could be a new starter or an existing developer. Using the following template will help to ensure each README is consistent, correct, short and actionable.
 
-- New starters to GDS
-- People joining a new team
-- People interested in our work
-- Developers on other teams
+Unless it helps someone to get started, any other content should be a separate file in the `docs/` directory for a repository. This also makes the content easier to discover in external search.
 
 ## Template for new READMEs
 
 ```markdown
 # App name
 
-One paragraph description and purpose.
+One or more paragraphs describing the app:
 
-## Screenshots (if there's a client-facing aspect of it)
+- What it's used for.
+- Any unusual dependencies it has.
 
-## Live examples (if available)
+## Live examples
 
-- [gov.uk/thing](https://www.gov.uk/thing)
+(Only applies to frontend apps.)
+
+A list of links to example pages rendered by the app.
 
 ## Nomenclature
 
-- **Word**: definition of word, and how it's used in the code
+A list of definitions for unusual terms in the code.
 
 ## Technical documentation
 
-Write a single paragraph including a general technical overview of the app.
-Example:
-
-This is a Ruby on Rails application that maps RESTful URLs onto a persistence
-layer. It's only presented as an internal API and doesn't face public users.
-
-### Dependencies
-
-- [alphagov/other-repo]() - provides some downstream service
-- [redis]() - provides a backing service for work queues
-
-### Running the application
+What goes here depends on the type of app.
 
-`./one-line-command.sh`
-
-Documentation for where the app will appear (default port, vhost, URL etc).
+See the links below for example content to put here.
 
 ### Running the test suite
 
-`./one-line-command.sh`
-
-Include any other edge cases, e.g parallel test runner in Whitehall
+Give one command to run all the tests, linting, etc.
 
-### Any deviations from idiomatic Rails/Go etc. (optional)
+You can also add other commands e.g. to run JS tests.
```

Comment:
Not sure I understand this? Is the point to be specific on the type of testing? E.g. "To run javascript tests, run the following... To run linting, run the following..."

Reply:
Yes, that's right. We could tweak it to be clearer perhaps. How about:

```
You can also suggest other commands e.g. to run JS tests, run ....
```

Reply:
works for me

Reply:
I've added a commit to change it to that 👍.